ui/admin/tests/helpers/selectors.js

```diff
@@ -18,6 +18,8 @@ export const FIELD_DESCRIPTION_VALUE = 'description';
 export const FIELD_PASSWORD = '[name=password]';
 export const FIELD_PASSWORD_VALUE = 'password';
 
+export const FIELD_ERROR = '.hds-form-error__message'; // Selects any error message on the DOM.
```

Comment:
Moved to common selectors because more than 1 domain uses it and more coming usage in different domains coming soon.